### ISSUE TITLE
Invalid service "dashboard.widget.mysql-max-used-connections": class "TYPO3\CMS\Dashboard\Widgets\DoughnutChartWidge   t" does not exist.

### ISSUE THREAD

**Issue by mediaessenz** [froemken/mysqlreport#97]
Extension is not working under TYPO3 11.5.30, but composer.json "require" says different: "^11.5.23 || ^12.2.0"
Also the ext_emconf.php says, it should be compatible with TYPO3 11.5.23 and up: 'typo3' => '11.5.23-12.5.99'

After install backend breaks with this error message:
```
Invalid service "dashboard.widget.mysql-max-used-connections": class "TYPO3\CMS\Dashboard\Widgets\DoughnutChartWidget" does not exist.
```

The class TYPO3\CMS\Dashboard\Widgets\DoughnutChartWidget is not available under TYPO3 11.5

**Comment by mediaessenz** [comment]
Found the issue: The package typo3/cms-dashboard was not installed.
Shoud be IMO a requirement in the composer.json of this extension.

**Comment by froemken** [comment]
Hi @mediaessenz 

good point. Of cause mysqlreport should also work without installed dashboard. I reopen the issue.
I have to find a solution how to load the dashboard class only if dashboard is installed. Maybe compilerpass.
I will have a look into that.

Thank you for reporting

Stefan

**Comment by froemken** [comment]
Hello @mediaessenz 

I have found a solution to remove all widgets from Symfony DI container, if EXT:dashboard is not installed. Works good in TYPO3 11 and 12.
I have just released version 3.0.3 to TER and composer.

Stefan
